.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,27 @@
+<!--- Provide a general summary of your changes in the Title above -->
+
+## Description
+<!--- Describe your changes in detail -->
+
+## Motivation and Context
+<!--- Why is this change required? What problem does it solve? -->
+<!--- If it fixes an open issue, please link to the issue here. -->
+
+## How Has This Been Tested?
+<!--- Please describe in detail how you tested your changes. -->
+<!--- Include details of your testing environment, and the tests you ran to -->
+<!--- see how your change affects other areas of the code, etc. -->
+
+## Types of changes
```

Comment:
Since we are using https://www.conventionalcommits.org/ we have type in PR title. But I would add to checklist that a developer marked it as breaking change if it breaks compatibility.

Reply:
Do you mean remove these items except for the one that indicates if it is a breaking change?

Reply:
We could add this to the Checklist section and remove the "Types of Changes":

>Breaking change:
>- [ ] Yes
>- [ ] No

Reply:
I think we don't need "Types of Changes" coz we specify it in PR title according to https://www.conventionalcommits.org/. Also, if three is breaking change you need to add "!" to PR title as well.
I was thinking to move it to "Checklist" as a reminder.

Probably, it's better to have a separate section "Breaking changes" where a developer should describe them if they are present (we need it for changelog).

Reply:
Dropped Types of Changes and added breaking changes section in b24930d